diagnostic.md

```diff
@@ -4,7 +4,15 @@ Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
+<meta content="Contribute to jquery-dom-diagnostic development by creating an account on GitHub." name="twitter:description">
 
+body class="logged-out   env-production linux vis-public fork">
+    <div id="js-pjax-loader-bar" class="pjax-loader-bar"></div>
+    <a href="#start-of-content" tabindex="1" class="accessibility-aid js-skip-to-content">Skip to content</a>
```

Comment:
What the questions was looking for was either class names, ID's or HTML elements that could be targeted via jQuery ( $ )